Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,15 +1,54 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
+        // *************private helpers**********
+        /// <summary>
+        /// Test that Array is neither null or empty.
+        /// </summary>
+        /// <returns>True if array can be manipulated by standard functions.</returns>
+        /// 
+        private static bool IsNullOrEmpty(int[] array)
+        {
+            return array == null || array.Length == 0;
+        }
+          
+        /// <summary>
+        /// Confirm thar the index to be modified exists.
+        /// </summary>
+        /// <returns>True if index is valid
+        /// 
+        private static bool IsIndexIsWithinRange(int[] array, string action, int index)
+        {
+            return action == "Remove" ? index < array.Length && index >= 0 : index <= array.Length && index >= 0;
+        }
+
+        //*************public array operations*************
```

Comment:
I hate comments like that, really not needed. If you want to divide code in segments, a better way would be:
``
#region `public array operations
// code goes here..
#endregion 
```
I personally avoid this, because it implies your file being too big. But it's up to your preference.